plugins/catalog-backend/src/database/DefaultProcessingDatabase.ts

```diff
@@ -113,7 +113,10 @@ export class DefaultProcessingDatabase implements ProcessingDatabase {
 
     // Delete old relations
     let previousRelationRows: DbRelationsRow[];
-    if (tx.client.config.client.includes('sqlite3')) {
+    if (
+      tx.client.config.client.includes('sqlite3') ||
+      tx.client.config.client.includes('mysql')
```

Comment:
oh really? i would have thought that the mysql driver supported RETURNING just fine?

Reply:
Unfortunately. When I put mysql as client this snippet throws the following warning (.returning() is not supported by mysql and will have no effect) and in another snippet it throws the following error (failed previous.relations.map is not a function type =plugin stack=TypeError: previous.relations.map is not a function)

Reply:
If you don't mind. I think that we could use the same solution, without returning, to all databases...

Reply:
@freben What do you think about it?